VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -1505,13 +1497,12 @@ public void Get_BirthRecordId()
         public void Set_BirthRecordState()
         {
             Dictionary<string, string> brs = new Dictionary<string, string>();
-            brs.Add("code", "MA");
-            brs.Add("system", "urn:iso:std:iso:3166:-2");
-            brs.Add("display", "Massachusetts");
+            Dictionary<string, string> iladdress = new Dictionary<string, string>();
+            brs.Add("code", "US-MA");
```

Comment:
Would having a test for just `MA` still be useful here as well just to test the additional codepaths that were added?